packages/main/src/plugin/libpodapi-enable/libpodapi-settings.ts

```diff
@@ -0,0 +1,22 @@
+/**********************************************************************
+ * Copyright (C) 2024 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+export enum LibpodapiSettings {
+  SectionName = 'libpodapi',
+  Enable = 'enable',
```

Comment:
it's misleading as for example we do use libpodAPI for listing containers and turning off this flag won't change anything

so we need to be more precise saying 'for list images' 'forImageList'